query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/select/Select.java

```diff
@@ -146,6 +146,8 @@ default Select orderBy(@NonNull String columnName, @NonNull ClusteringOrder orde
     return orderBy(CqlIdentifier.fromCql(columnName), order);
   }
 
+  @NonNull
+  Select orderBy(@NonNull Ann ann);
```

Comment:
I would consider adding here the direction (ASC, DESC) parameter. Currently we do not support DESC vector ordering, but this may be available in future and CQL syntax allows it.